lib/enjoi.js

```diff
@@ -156,6 +156,14 @@ module.exports = function enjoi(schema, options) {
     function object(current) {
         var joischema = Joi.object(resolveproperties(current));
 
+        if (Thing.isObject(current.additionalProperties)) {
+          joischema = joischema.keys(resolveproperties({properties: current.additionalProperties}));
+        }
+
+        if (current.additionalProperties) {
```

Comment:
I understand what you mean.  I've linked `enjoi` locally to ensure these changes don't break `swaggerize-express`, and when I didn't make this broader my app wouldn't come up.  I think it's because some of the additional properties in the swagger schema files are using other json-schema features that haven't been implemented yet.